src/main/java/io/jenkins/plugins/extendedsecuritysettings/ExtendedSecuritySettings.java

```diff
@@ -32,21 +32,30 @@
 import org.kohsuke.accmod.restrictions.NoExternalUse;
 import org.kohsuke.stapler.DataBoundSetter;
 
+import javax.annotation.CheckForNull;
 import javax.annotation.Nonnull;
+import java.util.HashSet;
+import java.util.Set;
 
 @Restricted(NoExternalUse.class)
 @Extension
 @Symbol("extendedSecuritySettings")
 public class ExtendedSecuritySettings extends GlobalConfiguration {
 
-    public static ExtendedSecuritySettings get() {
-        return GlobalConfiguration.all().get(ExtendedSecuritySettings.class);
+    public static @Nonnull ExtendedSecuritySettings get() {
+        ExtendedSecuritySettings settings = GlobalConfiguration.all().get(ExtendedSecuritySettings.class);
+        if (settings == null) {
```

Comment:
I'm curious if there's a reason this check is needed.

Reply:
Unfortunately, yes. The servlet filter was being executed before the extension list is assembled, so the first few requests have to live without a config until Jenkins finishes loading basically.

Reply:
Ah, interesting situation.

Reply:
As we are using 2.156 in this plugin and in Jenkins 2.119 we got a new method ([getInstance](https://github.com/jenkinsci/jenkins/commit/79ec930a8ef4f12d2f2574c50c2f42272e5c75fb#diff-e3f986a0854b9b724f4e12690cdeeac3R161)) that does exactly what you want there I imagine :)

Reply:
Thanks for the pointer, Wadeck! Updated.